src/engine/cache_engine.h

```diff
@@ -66,8 +66,10 @@ static inline bool ocf_req_cache_mode_has_lazy_write(ocf_req_cache_mode_t mode)
 
 bool ocf_fallback_pt_is_on(ocf_cache_t cache);
 
-struct ocf_request *ocf_engine_pop_req(struct ocf_cache *cache,
-		struct ocf_queue *q);
+bool ocf_seq_cutoff_check(ocf_core_t core, uint32_t dir, uint64_t addr,
+		uint64_t bytes);
```

Comment:
There is no such function in OCF. Please remove it from the header.

Reply:
Sorry, I don't know how it got here.